README.md

```diff
@@ -30,93 +24,80 @@ the BMC and the host of the machine(s) you're testing.
 
 ### Preparation ###
 
-**Machine Configuration**
+The target system will need to have an OS that can boot. That OS will
+need to have several things installed on it.
 
-Copy the bvt/op-machines-example.xml file and use its layout (specified
-in bvt/op-machines.xsd) to specify the machines in your test lab.
+A TODO item is to document what that is.
```

Comment:
When refactoring some of the test, I went down the route of requiring the machine to already have things like getscom/putscom/opal-gard installed rather than have the test system build it.

The idea being that the test suite shouldn't modify the system in any permanent way.

What if we moved the setup components (such as ensuring all the right things are installed) to a separate script? (perhaps a shell script to run on the host). That would make it a lot easier for someone to audit what they were about to install on their machine.